servers/pylsp/Dockerfile

```diff
@@ -1,20 +1,23 @@
-FROM alpine as build
+FROM alpine:3.22.1 as build
+
+ARG VERSION=1.12.0
+LABEL version="${VERSION}"
 
 RUN apk add --no-cache \
-    build-base \
-    python3 \
-    python3-dev \
-    py3-pip \
+    build-base=0.5-r3 \
+    python3=3.12.11-r0 \
+    python3-dev=3.12.11-r0 \
+    py3-pip=25.1.1-r0 \
   && python3 -m venv /venv
```

Comment:
Old packages are not kept in mirrors indefinitely. Once Python is upgraded (e.g.: to 3.12.12), this will break.

I assume you're pinning versions to ensure that distribution upgrade don't break builds. This really isn't necessary: Alpine 3.22 won't upgrade Python to an incompatible version (e.g.: to 3.12), so you don't need to pin packages like this.

I'm using `python3` as an example, but the same applies to all other packages.